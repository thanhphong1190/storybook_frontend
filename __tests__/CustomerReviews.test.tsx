import renderer from 'react-test-renderer'
import {
  CustomerReviews,
  CustomerReviewProps
} from '../src/components/customer-reviews/CustomerReviews'

const props: CustomerReviewProps = {
  title: 'Customer Reviews',
  reviews: [
    {
      customerName: 'Minh Song',
      date: '2/09/2018',
      stars: 5,
      review:
        'I signed up to the 12 week challenge as a way to stay on track and focus on my goals. Winning is obviously a factor too but I also wanted another option so I could continue to be motivated. The meal plans and group sessions created by Malcolm (who is honestly the best personal trainer I’ve ever worked with) has helped me learn new exercises, correct my form and know that I can eat without feeling guilty. I have lost 5 kilos from the beginning of the challenge until now.'
    },
    {
      customerName: 'Lisa Cao',
      date: '14/09/2017',
      stars: 4,
      review:
        'I signed up to the 12 week challenge as a way to stay on track and focus on my goals. Winning is obviously a factor too but I also wanted another option so I could continue to be motivated. The meal plans and group sessions created by Malcolm (who is honestly the best personal trainer I’ve ever worked with) has helped me learn new exercises, correct my form and know that I can eat without feeling guilty. I have lost 5 kilos from the beginning of the challenge until now.'
    }
  ]
}
it('CustomerReviews renders correctly', () => {
  const tree = renderer.create(<CustomerReviews {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
