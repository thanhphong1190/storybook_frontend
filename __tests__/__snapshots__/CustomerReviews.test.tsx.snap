// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerReviews renders correctly 1`] = `
<section
  className="CustomerReviewsstyled__Section-qg1ndp-2 dagLMj"
>
  <div
    className="CustomerReviewsstyled__Container-qg1ndp-3 doNCCK"
  >
    <div
      className="CustomerReviewsstyled__Title-qg1ndp-4 gHcGUZ"
    >
      Customer Reviews
    </div>
    <hr
      className="globalstyles__Underline-sc-1nc69wi-8 dVEiXG"
    />
    <div
      className="CustomerReviewsstyled__SliderWrapper-qg1ndp-5 eglaKU"
    >
      <div
        className="carousel-root CustomerReviewsstyled__StyledSlider-qg1ndp-0 jEzNHY"
        tabIndex={0}
      >
        <div
          className="carousel carousel-slider"
          style={
            Object {
              "width": "100%",
            }
          }
        >
          <button
            aria-label="previous slide / item"
            className="control-arrow control-prev"
            onClick={[Function]}
            type="button"
          />
          <div
            className="slider-wrapper axis-horizontal"
            style={Object {}}
          >
            <ul
              className="slider animated"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "MozTransform": "translate3d(-100%,0,0)",
                  "MozTransitionDuration": "350ms",
                  "MsTransform": "translate3d(-100%,0,0)",
                  "MsTransitionDuration": "350ms",
                  "OTransform": "translate3d(-100%,0,0)",
                  "OTransitionDuration": "350ms",
                  "WebkitTransform": "translate3d(-100%,0,0)",
                  "WebkitTransitionDuration": "350ms",
                  "msTransform": "translate3d(-100%,0,0)",
                  "msTransitionDuration": "350ms",
                  "transform": "translate3d(-100%,0,0)",
                  "transitionDuration": "350ms",
                }
              }
            >
              <li
                className="slide"
                onClick={[Function]}
              >
                <div
                  className="ReviewCardstyled__Container-a5510c-0 kYCWN"
                >
                  <div
                    className="ReviewCardstyled__StarsGroup-a5510c-1 hRnVvH"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                  <div
                    className="ReviewCardstyled__Date-a5510c-2 hbJXPP"
                  >
                    14/09/2017
                  </div>
                  <span
                    className="ReviewCardstyled__CustomerName-a5510c-3 yWrmM"
                  >
                    Lisa Cao
                  </span>
                  <div
                    className="ReviewCardstyled__ReviewText-a5510c-4 gqlMVI"
                  >
                    I signed up to the 12 week challenge as a way to stay on track and focus on my goals. Winning is obviously a factor too but I also wanted another option so I could continue to be motivated. The meal plans and group sessions created by Malcolm (who is honestly the best personal trainer Iâ€™ve ever worked with) has helped me learn new exercises, correct my form and know that I can eat without feeling guilty. I have lost 5 kilos from the beginning of the challenge until now.
                  </div>
                </div>
              </li>
              <li
                className="slide selected"
                onClick={[Function]}
              >
                <div
                  className="ReviewCardstyled__Container-a5510c-0 kYCWN"
                >
                  <div
                    className="ReviewCardstyled__StarsGroup-a5510c-1 hRnVvH"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                  <div
                    className="ReviewCardstyled__Date-a5510c-2 hbJXPP"
                  >
                    2/09/2018
                  </div>
                  <span
                    className="ReviewCardstyled__CustomerName-a5510c-3 yWrmM"
                  >
                    Minh Song
                  </span>
                  <div
                    className="ReviewCardstyled__ReviewText-a5510c-4 gqlMVI"
                  >
                    I signed up to the 12 week challenge as a way to stay on track and focus on my goals. Winning is obviously a factor too but I also wanted another option so I could continue to be motivated. The meal plans and group sessions created by Malcolm (who is honestly the best personal trainer Iâ€™ve ever worked with) has helped me learn new exercises, correct my form and know that I can eat without feeling guilty. I have lost 5 kilos from the beginning of the challenge until now.
                  </div>
                </div>
              </li>
              <li
                className="slide"
                onClick={[Function]}
              >
                <div
                  className="ReviewCardstyled__Container-a5510c-0 kYCWN"
                >
                  <div
                    className="ReviewCardstyled__StarsGroup-a5510c-1 hRnVvH"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                  <div
                    className="ReviewCardstyled__Date-a5510c-2 hbJXPP"
                  >
                    14/09/2017
                  </div>
                  <span
                    className="ReviewCardstyled__CustomerName-a5510c-3 yWrmM"
                  >
                    Lisa Cao
                  </span>
                  <div
                    className="ReviewCardstyled__ReviewText-a5510c-4 gqlMVI"
                  >
                    I signed up to the 12 week challenge as a way to stay on track and focus on my goals. Winning is obviously a factor too but I also wanted another option so I could continue to be motivated. The meal plans and group sessions created by Malcolm (who is honestly the best personal trainer Iâ€™ve ever worked with) has helped me learn new exercises, correct my form and know that I can eat without feeling guilty. I have lost 5 kilos from the beginning of the challenge until now.
                  </div>
                </div>
              </li>
              <li
                className="slide selected"
                onClick={[Function]}
              >
                <div
                  className="ReviewCardstyled__Container-a5510c-0 kYCWN"
                >
                  <div
                    className="ReviewCardstyled__StarsGroup-a5510c-1 hRnVvH"
                  >
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      className="svg-inline--fa fa-star fa-w-18 "
                      data-icon="star"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      style={Object {}}
                      viewBox="0 0 576 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                        fill="currentColor"
                        style={Object {}}
                      />
                    </svg>
                  </div>
                  <div
                    className="ReviewCardstyled__Date-a5510c-2 hbJXPP"
                  >
                    2/09/2018
                  </div>
                  <span
                    className="ReviewCardstyled__CustomerName-a5510c-3 yWrmM"
                  >
                    Minh Song
                  </span>
                  <div
                    className="ReviewCardstyled__ReviewText-a5510c-4 gqlMVI"
                  >
                    I signed up to the 12 week challenge as a way to stay on track and focus on my goals. Winning is obviously a factor too but I also wanted another option so I could continue to be motivated. The meal plans and group sessions created by Malcolm (who is honestly the best personal trainer Iâ€™ve ever worked with) has helped me learn new exercises, correct my form and know that I can eat without feeling guilty. I have lost 5 kilos from the beginning of the challenge until now.
                  </div>
                </div>
              </li>
            </ul>
          </div>
          <button
            aria-label="next slide / item"
            className="control-arrow control-next"
            onClick={[Function]}
            type="button"
          />
        </div>
        <div
          className="carousel"
        >
          <div
            className="thumbs-wrapper axis-vertical"
          >
            <button
              aria-label="previous slide / item"
              className="control-arrow control-prev control-disabled"
              onClick={[Function]}
              type="button"
            />
            <ul
              className="thumbs animated"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "MozTransform": "translate3d(0,0,0)",
                  "MozTransitionDuration": "350ms",
                  "MsTransform": "translate3d(0,0,0)",
                  "MsTransitionDuration": "350ms",
                  "OTransform": "translate3d(0,0,0)",
                  "OTransitionDuration": "350ms",
                  "WebkitTransform": "translate3d(0,0,0)",
                  "WebkitTransitionDuration": "350ms",
                  "msTransform": "translate3d(0,0,0)",
                  "msTransitionDuration": "350ms",
                  "transform": "translate3d(0,0,0)",
                  "transitionDuration": "350ms",
                }
              }
            />
            <button
              aria-label="next slide / item"
              className="control-arrow control-next control-disabled"
              onClick={[Function]}
              type="button"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
